contracts/v0.2/src/SubmissionProxy.sol

```diff
@@ -244,8 +247,8 @@ contract SubmissionProxy is Ownable {
     function submit(
         address[] memory _feeds,
         int256[] memory _answers,
-        bytes[] memory _proofs,
-        uint256[] memory _timestamps
```

Comment:
Is there a reason for changing the parameter order? If it is okay to keep the previous order, it'd be nice since there should also be updates in offchain codes. If you insist though, I think we can change this order and update offchain codes

Reply:
The reason is to keep components (`answer`, `timestamp`) used for proof generation closer to each other. I think it makes more sense this way. If we merge `answer` and `timestamp` to one parameter, then it would also look more like the current version.

Reply:
do you think merging answer and timestamp using abi.encodePakced will make it cheaper?

Reply:
I was planning to test it. I think that having 512 bytes for keeping time and value is a bit too much so both could be encoded with 256 bytes (64 bytes for timestamp and 192 for answer), but planned to test it in later PR.

Reply:
Actually, I think it could be even smaller than 256, maybe 128 bytes in total could be enough :)

Reply:
I think we should change the order of parameters anyway. I am not sure how it happened to be in this order.

Reply:
I won't insist so will approve

Reply:
Let's not modify anything in off-chain for now. Once, changes are settled, we can see where we stand 💪